aws-athena/views/default-vw_pin_universe.sql

```diff
@@ -1,146 +1,148 @@
-    -- or that are not 14 characters long
-    AND REGEXP_COUNT(par.parid, '[a-zA-Z]') = 0
-    AND LENGTH(par.parid) = 14
+-- Source of truth view for PIN location
+SELECT
+    -- Main PIN-level attribute data from iasWorld
+    par.parid AS pin,
+    SUBSTR(par.parid, 1, 10) AS pin10,
+    par.taxyr AS year,
+    REGEXP_REPLACE(par.class, '[^[:alnum:]]', '') AS class,
+    twn.triad_name,
+    twn.triad_code,
+    twn.township_name,
+    leg.user1 AS township_code,
+    REGEXP_REPLACE(par.nbhd, '([^0-9])', '') AS nbhd_code,
+    leg.taxdist AS tax_code,
+    NULLIF(leg.zip1, '00000') AS zip_code,
+
+    -- Centroid of each PIN from county parcel files
+    sp.lon,
+    sp.lat,
+    sp.x_3435,
+    sp.y_3435,
+
+    -- Corner lot indicator
+    lot.is_corner_lot AS ccao_is_corner_lot,
+
+    -- PIN locations from spatial joins
+    vwl.census_block_group_geoid,
+    vwl.census_block_geoid,
+    vwl.census_congressional_district_geoid,
+    vwl.census_county_subdivision_geoid,
+    vwl.census_place_geoid,
+    vwl.census_puma_geoid,
+    vwl.census_school_district_elementary_geoid,
+    vwl.census_school_district_secondary_geoid,
+    vwl.census_school_district_unified_geoid,
+    vwl.census_state_representative_geoid,
+    vwl.census_state_senate_geoid,
+    vwl.census_tract_geoid,
+    vwl.census_zcta_geoid,
+    vwl.census_data_year,
+    vwl.census_acs5_congressional_district_geoid,
+    vwl.census_acs5_county_subdivision_geoid,
+    vwl.census_acs5_place_geoid,
+    vwl.census_acs5_puma_geoid,
+    vwl.census_acs5_school_district_elementary_geoid,
+    vwl.census_acs5_school_district_secondary_geoid,
+    vwl.census_acs5_school_district_unified_geoid,
+    vwl.census_acs5_state_representative_geoid,
+    vwl.census_acs5_state_senate_geoid,
+    vwl.census_acs5_tract_geoid,
+    vwl.census_acs5_data_year,
+    vwl.cook_board_of_review_district_num,
+    vwl.cook_board_of_review_district_data_year,
+    vwl.cook_commissioner_district_num,
+    vwl.cook_commissioner_district_data_year,
+    vwl.cook_judicial_district_num,
+    vwl.cook_judicial_district_data_year,
+    vwl.ward_num,
+    vwl.ward_name,
+    vwl.ward_chicago_data_year,
+    vwl.ward_evanston_data_year,
+    vwl.chicago_community_area_num,
+    vwl.chicago_community_area_name,
+    vwl.chicago_community_area_data_year,
+    vwl.chicago_industrial_corridor_num,
+    vwl.chicago_industrial_corridor_name,
+    vwl.chicago_industrial_corridor_data_year,
+    vwl.chicago_police_district_num,
+    vwl.chicago_police_district_data_year,
+    vwl.econ_coordinated_care_area_num,
+    vwl.econ_coordinated_care_area_data_year,
+    vwl.econ_enterprise_zone_num,
+    vwl.econ_enterprise_zone_data_year,
+    vwl.econ_industrial_growth_zone_num,
+    vwl.econ_industrial_growth_zone_data_year,
+    vwl.econ_qualified_opportunity_zone_num,
+    vwl.econ_qualified_opportunity_zone_data_year,
+    vwl.econ_central_business_district_num,
+    vwl.econ_central_business_district_data_year,
```

Comment:
**question:** Shouldn't this be in a separate PR?

Reply:
I don't know why this is showing up as a change... it's in master, and I don't see a commit on this branch where I changed this?